tests/com/com_nrf24l01_tests.cpp

```diff
@@ -71,31 +71,42 @@ TEST_F(ComNRF24L01Tests, put_data_packet_failed_tx_timeout) {
 
   auto unit_under_test = std::make_unique<com::NRF24L01>(std::move(com_msg_buffer), std::move(com_nrf_core));
 
-  auto return_value = unit_under_test->PutDataPacket(types::PutDataTarget::TARGET_FRONT, default_data_packet_);
+  auto return_value = unit_under_test->PutDataPacket(types::PutDataPacketTarget::TARGET_FRONT, default_data_packet_);
   ASSERT_EQ(return_value, types::DriverStatus::TIMEOUT);
 }
 
 TEST_F(ComNRF24L01Tests, put_data_packet_failed_inconsistent_target) {
   auto com_msg_buffer = std::make_unique<NiceMock<com::ComMessageBuffer>>();
   auto com_nrf_core = std::make_unique<NiceMock<com::NRF24L01Core>>();
   types::ComDataPacket mock_packet{
-      .type = types::ComPacketType::TELEMETRY_PACKET,
-      .target = types::PutDataTarget::TARGET_FRONT};
+      .type = types::ComDataPacketType::TELEMETRY_PACKET,
```

Comment:
this is c99 "designated initializers" feature and just compiles with gcc because they have left it in for c++ as well. This technically is not part of the C++ specification

Reply:
it would be better if you change it, in order to stay compliant with cpp standard

Reply:
Done.